app/controllers/api/v2/helm_infrastructures_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V2::HelmInfrastructuresController < Api::V2::BaseController
```

Comment:
please create test for this one under this folder https://github.com/BaritoLog/BaritoMarket/blob/master/spec/requests/api/v2/app_groups_spec.rb